### PR TITLE
Update layout

### PR DESCRIPTION
**Pull request by tmskrtsz**

### CODE REVIEW

File: common/Footer.tsx [hunk 1]
```diff
@@ -23,84 +23,7 @@ export const Footer = ({
...
-            </a>
-            <a
-              href="https://discord.com/invite/byq6uNTugq"
-              className="text-gray-400"
-            >
-              Discord
-            </a>
-          </span> */}
-        </div>
-      </div>
```
Comment (jpbogle) [inline common/Footer.tsx:103]: not sure we want to remove this for everyone, happy to accept a config option for a pool that makes it hidden

File: common/Header.tsx [hunk 2]
```diff
@@ -1,176 +1,166 @@
...
+  DialectContextProvider,
+  DialectThemeProvider,
+  DialectUiManagementProvider,
+  DialectWalletAdapter,
+  NotificationsButton,
+} from '@dialectlabs/react-ui'
+import { EnvironmentType } from 'types/environment'
+import { ClockdriftWarning } from 'components/ClockdriftWarning'
+
+const walletToDialectWallet = (
```
Comment (jpbogle) [inline common/Header.tsx:36]: cool!

File: hooks/useNotifications.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+  function notify({ message, type, description }: Omit<Notification, 'id'>) {
+    const lastElement = notifications[notifications.length - 1]
+    const lastId = lastElement ? lastElement.id + 1 : 0
+
+    add({ message, description, type, id: lastId })
+  }
+
+  return { notify, notifications, remove }
+}
```
Comment (jpbogle) [inline hooks/useNotifications.tsx:56]: hmmm kinda not sure we want to use zustand here as yet another state management library, was there any issues with the existing notifications? We prefer using this one now so maybe we can just update to that just havent gotten around to updating it here

https://github.com/cardinal-labs/cardinal-rental-ui/blob/main/common/Notification.tsx

**Comment by gchatz22** [comment]
could you pull the latest changes from main into your branch

**Comment by gchatz22** [comment]
could you pull the latest changes from main into your branch

**Comment by gchatz22** [comment]
hey any updates here? you need to rebase off latest main

**Comment by gchatz22** [comment]
closing due to inactivity and non-responsiveness, feel free to open a new PR if you want with the changes mentioned above
\

**Pull request closed without merging**